client/packages/programs/src/api/api.ts

```diff
@@ -401,3 +407,59 @@ export const getProgramEventQueries = (sdk: Sdk, storeId: string) => ({
     throw new Error('Error querying program events');
   },
 });
+
+export type ContactTraceListParams = {
+  sortBy?: SortRule<ContactTraceSortFieldInput>;
+  filterBy?: ContactTraceFilterInput;
+};
+
+export const getContactTraceQueries = (sdk: Sdk, storeId: string) => ({
+  contactTraces: async ({
```

Comment:
I renamed them. Find it confusing to name them like this though. For example, its usually used like:

```ts
  const api = useContactTraceApi();
  return useMutation(api.insert, {
```

The api endpoint is `insertContactTrace` though and not insert... Find the api folder quite hard to read/write since you need to go through so many files to get to the part that actually calls the api, having the original endpoint name helps a little bit.

Reply:
sure - up to you then. 
another suggestion is to be more specific with the local var

```
const contactTraceApi = useContactTraceApi();
  return useMutation(contactTraceApi.insert, {
```

but it's fine either way. objection withdrawn :)